### PR TITLE
Changed agent to bot and did modifications to match requirements

### PR DESCRIPTION
**Pull request by Tdrachev**
Updated bot to match haseeb requests

### CODE REVIEW

**Review by dikel — APPROVED** [review]
LGTM

File: forta-admin-bot-scanner-disable/src/agent.js [hunk 1]
```diff
@@ -42,41 +43,46 @@ function provideHandleTransaction(getContractNames, getContractAddresses) {
...
-                contractName,
-              },
-            })
-          );
+          if (disabledBy == "ADMIN")
+            findings.push(
+              Finding.fromObject({
+                name: "Forta scanner disabled",
+                description: `Forta scanner disabled, ScannerId: ${scannerIdAddress}`,
+                alertId: "FORTA-SCANNER-DISABLED",
```
Comment (haseebrabbani) [inline forta-admin-bot-scanner-disable/src/agent.js:54]: lets make this FORTA-SCANNER-ADMIN-DISABLED

File: forta-admin-bot-scanner-disable/src/agent.js [hunk 2]
```diff
@@ -42,41 +43,46 @@ function provideHandleTransaction(getContractNames, getContractAddresses) {
...
-                contractName,
-              },
-            })
-          );
+          if (disabledBy == "ADMIN")
+            findings.push(
+              Finding.fromObject({
+                name: "Forta Bot disabled",
+                description: `Forta Bot disabled, agentId: ${agentIdAddress}`,
+                alertId: "FORTA-BOT-DISABLED",
```
Comment (haseebrabbani) [inline forta-admin-bot-scanner-disable/src/agent.js:75]: FORTA-BOT-ADMIN-DISABLED*

**Review by stoqnkpL — APPROVED** [review]
LGTM!

**Pull request merged**